### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by samuellefever**
Remove duplicated text

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -184,13 +184,6 @@ Use the following pattern:
 </record>
 ```
 
-These rules should be followed even when a view is inherited, the module name
-prevents xid conflicts. In the case where a view inherited has a name which
-does not follow the guidelines set above, prefer naming the inherited view
-after the original over using a name which follows the guidelines. This eases
-looking up the original view and other inheritance if they all have the same
-name.
-
```
Comment (dreispt) [inline CONTRIBUTING.md]: I'm sorry, I don't see the duplication.

**Comment by yvaucher** [comment]
:-1: I don't see the duplication either

File: CONTRIBUTING.md (hunk 1 above)
Comment (guewen) [inline CONTRIBUTING.md]: @dreispt look at the end of this section https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#security-view-and-action and at the beginning of the next one

**Comment by guewen** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**